src/hooks/useHorizontalScroll.ts

```diff
@@ -0,0 +1,25 @@
+import { useRef, useEffect, WheelEvent } from "react";
+
+export function useHorizontalScroll() {
+  const elRef = useRef();
+  useEffect(() => {
+    const el = elRef.current;
+    if (el) {
+      const onWheel = (e: WheelEvent<HTMLUListElement>) => {
+        if (e.deltaY == 0) return;
+        e.preventDefault();
+      // @ts-expect-error - Do not hace access to the right element
```

Comment:
The reason for this expect-error is that we do not have access to the native web API to get the correct events

Is ugly but necessary given the circumstances